pkg/clients/kubernetes.go

```diff
@@ -0,0 +1,48 @@
+package clients
+
+import (
+	logging "github.com/op/go-logging"
+	restclient "k8s.io/client-go/rest"
+
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+)
+
+func createClientConfigFromFile(configPath string) (*restclient.Config, error) {
+	clientConfig, err := clientcmd.LoadFromFile(configPath)
+	if err != nil {
+		return nil, err
+	}
+
+	config, err := clientcmd.NewDefaultClientConfig(*clientConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
+	if err != nil {
+		return nil, err
+	}
+	return config, nil
```

Comment:
:bulb: Ok so this file is from the previous PR. Finally understood what's going on. I find the dependent PRs much harder to review. I would've preferred a clients feature to do all of the cleanup at once.